### PR TITLE
Add guide on how to add assets for the back end (#50)

### LINKED ISSUE #50: Cookbook: add custom CSS to the back end

### PR DESCRIPTION
**Pull request by fritzmg**
Closes #50 
Depends on #332

### CODE REVIEW

File: docs/dev/guides/adding-back-end-assets.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+    public function onKernelRequest(RequestEvent $e): void
+    {
+        $request = $e->getRequest();
+
+        if ($this->scopeMatcher->isBackendRequest($request)) {
+            if (!isset($GLOBALS['TL_JAVASCRIPT'])) {
+                $GLOBALS['TL_JAVASCRIPT'] = [];
+            }
+
```
Comment (ausi) [inline docs/dev/guides/adding-back-end-assets.md]:
```suggestion
```

Setting `$GLOBALS['TL_JAVASCRIPT']` to an empty array is not necessary as it is done automatically by PHP with the next line.

File: docs/dev/guides/adding-back-end-assets.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+class FilesOnLoadCallbackListener implements ServiceAnnotationInterface
+{
+    public const CSS_FILE = 'files/backend/tl_content.css';
+
+    public function __invoke(): void
+    {
+        if (!isset($GLOBALS['TL_CSS'])) {
+            $GLOBALS['TL_CSS'] = [];
+        }
+
```
Comment (ausi) [inline docs/dev/guides/adding-back-end-assets.md]:
```suggestion
```

Same here.

**Review by ausi — APPROVED** [review]

File: docs/dev/guides/adding-back-end-assets.md (hunk 1 above)
Comment (fritzmg) [reply docs/dev/guides/adding-back-end-assets.md]: Hm, wouldn't it cause a notice?

Comment (ausi) [reply docs/dev/guides/adding-back-end-assets.md]: No, see https://3v4l.org/AouCI

**Pull request merged**